spec/calculator.spec.js

```diff
@@ -1,24 +1,31 @@
-'use strict';
+(function (factory) {
+  if (typeof require === 'function' && typeof exports === 'object' && typeof module === 'object') {
+    module.exports = factory(require('chai'), require('../Calculator'));
```

Comment:
This code will run if the test is being run from Node JS. Notice how it uses `require(...)` to bring in chai and Calculator so that it can pass them to our tests via the factory function.